mitosheet/src/mito/components/toolbar/Toolbar.tsx

```diff
@@ -53,50 +46,16 @@ export const Toolbar = (
         closeOpenEditingPopups: () => void
     }): JSX.Element => {  
 
+    const [currentTab, setCurrentTab] = React.useState<string>('Home');
```

Comment:
if you make it <ToolbarTab | undefined>, then it's really natural to express the closing operation (it's just undefined)